### ISSUE TITLE
Steller Foundation Staking scam

### ISSUE THREAD

**Issue by gcr47** [stellar-expert/stellar-expert-explorer#762]
Person with public address GBENQQUDHRSE4FESH66HOVX2TZLWMURZJTNDJX5ZNTDVFUPAFW3UTP2U

Please return my 180,498 XLM that was taken. It was my life savings. I am crushed.

**Comment by CryptoHobbyist** [comment]
Hello @gcr47 

I am so sorry to hear of the theft. You first need to report this to law enforcement. 

NEVER GIVE ANYONE YOUR PRIVATE KEY. Scammers will try to approach victims pretending to be tech support to help. Real tech support will not need your private key to investigate or "return" stolen assets. 

Stellar Expert is only a Stellar blockchain explorer to help people see transactions on the Stellar Blockchain. We have no way to recover or stop transactions but can be used to help trace transactions.

You should directly report to these agencies that investigate scams:

First contact local police and get a police report and case number. Realize that the local authorities can not really help you with the cyber crime but a report will be needed to get any help from almost every crypto exchange and legal agency that can help you.. One of these should be enough, they are connected with cyber crime investigations.

https://www.ic3.gov/
http://secretservice.gov/investigation/cyber
https://www.fbi.gov/how-we-can-help-you/safety-resources/scams-and-safety
https://reportfraud.ftc.gov/
https://www.sec.gov/tcr

It looks like some how someone got your private key to start this series of transvers of the XLM. Any information you can give to authorities about how this might have happened would be very helpful.

Information that will be helpful for the authorities:

GCUSYYCIIKXXFRONYPJWNUA3J2WN4HII66UZUMLOELKOQRYT4PSJ5T5O (created/funded with Binance) 
Transaction ID 4d3f7afde0182fd3d989ae20ffb249cdf1e97083230966cfb8cf00ac7999f3fd back on 2019-1-18
https://stellar.expert/explorer/public/tx/4d3f7afde0182fd3d989ae20ffb249cdf1e97083230966cfb8cf00ac7999f3fd

GCUSYYCIIKXXFRONYPJWNUA3J2WN4HII66UZUMLOELKOQRYT4PSJ5T5O (where money was sent)
created/funded GC37XZCWIM4AKSM3U2GMWBB4GIJFDDANTU53AM3CQUJOJ43AYIUE6OL4 wallet with all the funds. As seen here: Transaction ID 9c9fc832bfc81e7b20d18d24888e6f13d391f622055edb431a3063075a84f518  2023-06-21 16:29:17 UTC
https://stellar.expert/explorer/public/tx/9c9fc832bfc81e7b20d18d24888e6f13d391f622055edb431a3063075a84f518

GC37XZCWIM4AKSM3U2GMWBB4GIJFDDANTU53AM3CQUJOJ43AYIUE6OL4 (then sent the money)
created/funded GBHPXJWO3CZOQJ7QZZBP62U6NQZW6MPBKO2QSY6BUZZ66GP6PP5OR457
wallet with all the funds. As seen here: Transaction ID fbf7613b2b271694e7d6b1d171e9a5db504e2f12f2a4c83e7849cddb9ccca1a5
https://stellar.expert/explorer/public/tx/fbf7613b2b271694e7d6b1d171e9a5db504e2f12f2a4c83e7849cddb9ccca1a5

GBHPXJWO3CZOQJ7QZZBP62U6NQZW6MPBKO2QSY6BUZZ66GP6PP5OR457 (then sent the money)
created/funded GBENQQUDHRSE4FESH66HOVX2TZLWMURZJTNDJX5ZNTDVFUPAFW3UTP2U
wallet with all the funds. As seen here: Transaction ID 609551bf0c06052cae6d3045d1e4d8783c80efe24a81b3aff2f225ba45bc4da5
https://stellar.expert/explorer/public/tx/609551bf0c06052cae6d3045d1e4d8783c80efe24a81b3aff2f225ba45bc4da5

Which still is holding it as of this quick investigation.

**Comment by gcr47** [comment]
Thank you CryptoHobbyist, it was a link from Binance that brought me to the Stellar Foundation blog, which looked very legit. I then sent the XLM to the staking site from the Stellar Foundation blog, all looked very legit. But I created a private key and entered it to log into the staking site. That is how they must have gotten ahold of the funds. These guys are very good at what they do. It looked very legitimate.

**Comment by CryptoHobbyist** [comment]
@gcr47 
You should gather as much information about the scam websites you were tricked into sending your key to. This site can help you see where the domain was registered and what servers they work off of. You might also contact those to report the crime (but they will require a police report and most likely one of the agencies to contact them about specifics) and maybe get the site shut down and/or set up a detailed logging of events on their end from this site.

Over the last few years https://stellar.expert/explorer/public/account/GCUSYYCIIKXXFRONYPJWNUA3J2WN4HII66UZUMLOELKOQRYT4PSJ5T5O?account[]=GCUSYYCIIKXXFRONYPJWNUA3J2WN4HII66UZUMLOELKOQRYT4PSJ5T5O
account has had multiple know scammers send small payments to you. 

You might have been tricked into giving one of these scammers your private key in some way and they have played the long con and only yesterday took all the assets.

You should also watch the current holder and when they move the assets to an exchange (requiring memos) then contact that exchange with the memo number (account on that exchange) and the above information. Occasionally some exchanges will freeze accounts on their exchange and wait for the police to contact them. Unfortunately there are some very shady exchanges that do not have any kind of KYC (know your customer) to help the police make an arrest even when their is no doubt that a crime has occurred.

I do not want to get your hopes up. You most likely have to live with the tragic results of having such a theft.

Rob

**Comment by gcr47** [comment]
Thanks again. Filed a complaint. Will continue to watch the account with my stolen XLM everyday to see what happens.

**Comment by CryptoHobbyist** [comment]
@gcr47 

Please DM any new changes. I'll leave this up a week or so and then close the issue. I would still like to have an update when something happens.

Looking Towards The Future With You,
Rob

**Comment by gcr47** [comment]
Will do. Thanks for the help Rob!

**Comment by gcr47** [comment]
By the way, first time here. How exactly do I DM you directly in the future?

**Comment by CryptoHobbyist** [comment]
@gcr47 

Good point, feel free to use thecryptohobbyist@gmail.com

**Comment by piposan** [comment]
> Person with public address that is holding my stolen XLM: GBENQQUDHRSE4FESH66HOVX2TZLWMURZJTNDJX5ZNTDVFUPAFW3UTP2U
> 
> Please return my 180,498 XLM that was taken. It was my life savings. I am crushed.
> 
> My public XLM address from which these XLM was taken is: GCUSYYCIIKXXFRONYPJWNUA3J2WN4HII66UZUMLOELKOQRYT4PSJ5T5O
> 
> Can someone from Stellar help?! I am so devastated

Had you received an email from Binance regarding Staking Rounds ? Simular to this one ?

<img width="989" alt="Screenshot 2023-06-24 at 09 48 30" src="https://github.com/stellar-expert/stellar-expert-explorer/assets/29426190/d90d5f18-e0e8-4aed-a69c-f92dcef7ba45">

**Comment by gcr47** [comment]
Yes. It was exactly like that email, which sent me to the Stellar
Foundation network blog to stake my coins

**Comment by gcr47** [comment]
Thank you. Will keep it touch

**Comment by Conspiracy12** [comment]
> Thank you. Will keep it touch
> […](#)

I had lost a lot of coins in some scam case just like this thanks to the best hacker in the world who helped me recover all my funds,message him on telegram his telegram username is https://t.me/Blackhathacker110

**Comment by HenryEnrique** [comment]
> Person with public address that is holding my stolen XLM: GBENQQUDHRSE4FESH66HOVX2TZLWMURZJTNDJX5ZNTDVFUPAFW3UTP2U
> 
> Please return my 180,498 XLM that was taken. It was my life savings. I am crushed.
> 
> My public XLM address from which these XLM was taken is: GCUSYYCIIKXXFRONYPJWNUA3J2WN4HII66UZUMLOELKOQRYT4PSJ5T5O
> 
> Can someone from Stellar help?! I am so devastated

Be careful out there I was scammed of all my Bitcoin through something like this,it took me even months to figure out how to solve it,then I came across this great lawyer we did some little job and everything was reversed to me immediately his one telegram message him and talk about yours https://t.me/Cryptolawyer4rensic
